dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -18,9 +18,17 @@
 
 _tf_plugins = glob.glob(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'libdali_tf*.so'))
 _dali_tf_module = None
-for _libdali_tf in _tf_plugins:
+# move plugin build aganist nightly to the end of list
+_processed_tf_plugins = list(filter(lambda x: 'nightly' not in x, _tf_plugins))
```

Comment:
How about:
`_processed_tf_plugins = list(filter(lambda x: 'nightly' not in x, _tf_plugins)) + list(filter(lambda x: 'nightly' in x, _tf_plugins))`